Stylesheets/Components/image.css

```diff
@@ -0,0 +1,17 @@
+.sq-img{
+    height:10rem;
+    width:10rem;
+    /* border-radius: 50%; */
```

Comment:
remove comments

Reply:
Thanks for pointing it out will remove the comments.